assets/js/components/BatchEdit/ModalRemove.js

```diff
@@ -90,8 +92,8 @@ export default function BatchEditAboutModalRemove({
           </div>
           <footer>
             <div className="buttons is-right">
-              <Button isText onClick={closeModal}>
-                Close
+              <Button isLight onClick={closeModal}>
+                Save &amp; close
```

Comment:
Can we make this capital `Save & Close`?

Reply:
@divyareddynu I guess I was looking at Github here as an example for the buttons and how they handle capitilization.  I kind of like the style of having buttons read as mini sentences, if that's ok by you.

Reply:
Cool, marking this as resolved.